tubemq-agent/pom.xml

```diff
@@ -0,0 +1,321 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <groupId>org.apache.tubemq</groupId>
+    <artifactId>tubemq-agent</artifactId>
+    <packaging>pom</packaging>
+    <version>0.0.1-SNAPSHOT</version>
+    <modelVersion>4.0.0</modelVersion>
+
+    <modules>
+        <module>agent-common</module>
+    </modules>
+
+    <properties>
+        <awaitility.version>4.0.3</awaitility.version>
+        <bytebuddy.version>1.10.10</bytebuddy.version>
+        <common.io>2.6</common.io>
+        <common.lang3.version>3.10</common.lang3.version>
+        <commons.cli.version>1.4</commons.cli.version>
+        <dbutils.version>1.7</dbutils.version>
+        <encoding>UTF-8</encoding>
+        <gson.version>2.8.5</gson.version>
+        <guava.version>12.0.1</guava.version>
+        <jdk.version>1.8</jdk.version>
+        <log4j2.version>2.13.1</log4j2.version>
+        <mockito.version>3.3.3</mockito.version>
+        <plugin.assembly.version>3.2.0</plugin.assembly.version>
+        <plugin.compile.version>3.8.1</plugin.compile.version>
+        <rocksdb.version>6.8.1</rocksdb.version>
+        <sleepycat.version>18.3.12</sleepycat.version>
+        <slf4j.version>1.7.30</slf4j.version>
+        <unit.version>4.13</unit.version>
+        <bussdk.version>1.2.17</bussdk.version>
+        <common.lang.version>2.4</common.lang.version>
+        <spring.version>2.5.6</spring.version>
+        <oro.version>2.0.8</oro.version>
+        <aviator.version>2.2.1</aviator.version>
+        <avro.version>1.7.2</avro.version>
+
+        <netty.version>3.8.0.Final</netty.version>
+        <snappy.version>1.0.4.1</snappy.version>
+        <protobuf.version>2.5.0</protobuf.version>
+        <httpclient.version>4.5.13</httpclient.version>
+        <fastjson.version>1.2.68</fastjson.version>
+        <sleepycat.version>6.4.9</sleepycat.version>
```

Comment:
The 6.4.9 version of sleepycat is not the Apache v2 version

Please check the LICENSE of the code and the dependent component packages before incorporating the code on a large scale.